.travis.yml

```diff
@@ -24,3 +24,5 @@ jobs:
     - stage: coverage
       python: '3.6'
       script: codecov
+    - stage: test
+      python: '3.9'
```

Comment:
This should move up before coverage otherwise 3.9's coverage will not be included in the data sent to `codecov`.